package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "meilisearch",
-  "version": "0.34.1",
+  "version": "0.34.2-tokenizer-customization.0",
```

Comment:
@bidoubiwa do we really need to release a meilisearch-js beta? Should we test it with instant-meilisearch?

Reply:
Instant-meilisearch is not impacted by this new change!

I released it as it is part of the prototyping process, so that people can try it out. If you think it's not necessary, it is less work for us to not release it hehe

Reply:
Maybe the process is too complex for what we want to do
We just want to test the prototype is working with the library. Can we have a risk of not releasing it? I mean, can we have a difference of usage between using it from source, or by using the NPM package?

From my POV, releasing a NPM package is useful only if you want to use it with another library, like IS? Am I right?
For example, we will not release a beta for the PHP integration. 
I also can understand sometimes it's useful for the users to test the beta version of the library however, especially for front end oriented features!

So I would say: let's show the full process to @atoulmet (so release the beta version) even if it's not necessary, but in the future, we would rather go to simplicity and do only what is useful for us: ensuring the library can be compatible with the prototype.

Reply:
> I mean, can we have a difference of usage between using it from source, or by using the NPM package?

No reasons there should be any difference.


> From my POV, releasing a NPM package is useful only if you want to use it with another library, like IS? Am I right?

This would be the biggest reason. Another reason we had in the past is if we wanted users to be able to try the feature in their javascript environment without having to use a HTTP lib.

> f you agree, let's show the full process to @atoulmet (so release the beta version) even if it's not necessary, but in the future, we would rather go to simplicity and do only what is useful for us: ensuring the library can be compatible with the prototype.

i completely agree!